testing/gcp_util.py

```diff
@@ -100,19 +102,22 @@ def endpoint_is_ready(url, wait_min=15):
       resp = requests.request(
           "GET",
           url,
-          headers={
-              "Authorization":
-              "Bearer {}".format(google_open_id_connect_token)
-          },
-          verify=False)
+          verify=False) if use_basic_auth else requests.request(
```

Comment:
Is there a typo here? This doesn't looks like valid syntax

Reply:
yes, it is valid - this is like Python version of syntax sugar.

Reply:
Google style guide suggests eschewing complex "list comprehensions". If it doesn't fit on one line its a good indicator its too complicated.

Can we use a standard multi-line if/else statement?

Reply:
done.